WonderBewegung.py

```diff
@@ -0,0 +1,261 @@
+from WonderWoerter import *
+from WonderTexte import *
+
+class Bewegung(object):
+    def __init__(self, startUmgebung, wonderText:WonderText, wortVergleicher:WortVergleicher):
```

Comment:
Ich finde die Benutzung von TypeHints nicht schlecht. Dies macht es mir als Reviewerin (oder Programmiererin) einfacher zu verstehen, was die Methoden machen sollten. Ich würde die Benutzung allerings konsistent halten: d.h. wenn du dich dafür entscheidest, würde ich sie überall benutzen, nicht nur an manchen Stellen. Hier zb.
```suggestion
    def __init__(self, startUmgebung: WonderUmgebung, wonderText:WonderText, wortVergleicher:WortVergleicher) -> Bewegung:
```
(und dann auch wirklich möglichst alles annotieren).
Beachte auch, dass diese vom Interpreter nicht evaluiert/überprüft werden, sondern lediglich als Hint für die Leserin funktionieren. (Allerdings gibt es anscheinend 3rd party tools, die das TypeChecking machen können.) Siehe auch https://peps.python.org/pep-0484/ für mehr Infos.